lib/models/event.js

```diff
@@ -142,5 +145,13 @@ module.exports.MatrixEvent.prototype = {
      */
     isState: function() {
         return this.event.state_key !== undefined;
+    },
+
+    getPushActions: function(client) {
```

Comment:
Hmm - adding a constructor arg to MatrixEvent for something that specific doesn't seem ideal, and I'm not too keen on complicating it with closures. I'd be in favour of either just adding the creating client as an ctor arg to MatrixEvent so it can defer back to the client for doing this or other similar things, or leave the MatrixEvent model class as pure data and shift any more complex stuff like this out to MatrixClient.getPushActionsForEvent(event) - what do you think?